specs/passenger_spec.rb

```diff
@@ -67,4 +73,45 @@
       end
     end
   end
+
+  describe "net_expenditures" do
+    before do
+      # TODO: you'll need to add a driver at some point here.
+      @passenger = RideShare::Passenger.new(
+        id: 9,
+        name: "Merl Glover III",
+        phone_number: "1-602-620-2330 x3723",
+        trips: [],
+      )
+      trip1 = RideShare::Trip.new(
+        id: 8,
+        passenger: @passenger,
+        start_time: "2016-08-08",
+        end_time: "2016-08-09",
+        cost: 10,
+        rating: 5,
+        driver_id: 2,
+      )
+      trip2 = RideShare::Trip.new(
+        id: 9,
+        passenger: @passenger,
+        start_time: "2016-08-10",
+        end_time: "2016-08-11",
+        cost: 15,
+        rating: 4,
+        driver_id: 2,
+      )
+
+      @passenger.add_trip(trip1)
+      @passenger.add_trip(trip2)
+    end
+
+    it "each passengers total trip costs" do
+      expect(@passenger.net_expenditures).must_equal 25
+    end
+
+    it "total amount of time each passenger has spent on trips" do
+      expect(@passenger.total_time_spent).must_equal 172800
```

Comment:
You should also have tests for a passenger without any trips.

You also have no tests for passengers with in-progress trips.